packages/next/build/polyfills/fetch/whatwg-fetch.js

```diff
@@ -0,0 +1,10 @@
+/* globals self */
+const fetch = {
```

Comment:
This is a confusing name IMO, as it's a reserved ("confusing") global.

Let's rename?

```suggestion
const stub = {
```